include/deal.II/lac/lapack_support.h

```diff
@@ -26,7 +26,7 @@ DEAL_II_NAMESPACE_OPEN
 namespace LAPACKSupport
 {
   /**
-   * Most LAPACK functions change the contents of the matrix applied to to
```

Comment:
I read this a few more times and it appears the original is actually correct, just a little difficult to read: the first clause is "Most LAPACK functions change the contents of the matrix applied to" and the second is "to something which is not a matrix anymore".